src/components/Routes/Routes.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { AdviceContainer } from '../AdviceContainer/AdviceContainer';
+import { AppContainer } from '../AppContainer/AppContainer';
+import { ComplimentContainer } from '../ComplimentContainer/ComplimentContainer';
+
+export interface RoutesProps {
```

Comment:
Nice work defining interfaces for your routes!  Potentially maybe some of your interfaces could get moved into their own file and then imported where they are needed?